### ISSUE TITLE
Mathbran format to latex or MathMarkupLangugae conversion and vice-versa

### ISSUE THREAD

**Issue by aifylabs** [JohnDTill/MathBran#5]
Hi, 
Is there a way to save the typed equation in tex format or MathML format ? Such conversion would allow user to edit the equations even if we export the whole document in word format.

**Comment by JohnDTill** [comment]
Yes, [YAWYSIWYGEE](https://github.com/JohnDTill/YAWYSIWYGEE) includes an option to copy the selection as TeX. It's a feature in the desktop app, although I never bothered converting it to the web app since it was implemented to populate your clipboard, which isn't allowed without an explicit ctrl-c in a browser.

These days I've stopped work on Mathbran/YAWYSIWYGEE in favour of Forscape though
